frontend/public/style/_common.scss

```diff
@@ -202,6 +202,18 @@ input[type=number]::-webkit-outer-spin-button {
   background-color: #D64456;
 }
 
+.co-action-divider {
+  display: none;
```

Comment:
Let's not assume that links will stack and the divider should be hidden in all cases. That really depends on context.

Reply:
So it sounds like we shouldn't try to have a common `co-action-divider` class then?  It sounds like I should just use a common color variable in both `pairs-list__action-divider` and `.co-toolbar__item--divider`. rather than trying to use `co-action-divider` in both places.

Reply:
I still think it's the right thing to do.

I'd leave the common rule and keep the pairs rule with the media queries.

Reply:
If we don't have a common rule for the dividers, the styles will drift from page to page.

Reply:
> I'd leave the common rule and keep the pairs rule with the media queries.

Ok, will do, but should the resource log hide the divider at mobile?

Reply:
Ok, `co-action-divider` just has the color, `pairs-list__action-divider` hides the divider at mobile.  I can add `.co-toolbar__item--divider` to also hide the divider at mobile if that is what we decide.